front/src/components/filters/range-filter/RangeFilter.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { RangeFilter } from "../../../services/filterService";
+
+interface RangeFilterProps {
+    filter: RangeFilter;
+}
+
+const RangeFilter = ({filter}: RangeFilterProps) => {
+    console.log(filter);
+    console.log(filter.name);
+    return <div>
```

Comment:
we can use <></> here

Reply:
we'll use some classnames here probably, thats why I use div

Reply:
Ok,np